### ISSUE TITLE
Timeout connecting to T8400 Devices

### ISSUE THREAD

**Issue by shaunkutch** [fuatakgun/eufy_security#332]
**UPDATE:**
It appears that the add-on is not connecting to my T8400 devices. They show up in the integration but do not respond to services.
```
2022-05-17 07:01:44.546  INFO  Timeout connecting to station T8400xxxxxxxxxxx
2022-05-17 07:03:55.392  ERROR P2PClientProtocol.lookup(): station: T8400xxxxxxxxxxx - All address lookup tentatives 
```

Possible related issues:
https://github.com/fuatakgun/eufy_security/issues/306
https://github.com/fuatakgun/eufy_security/issues/318
https://github.com/fuatakgun/eufy_security/issues/326

## Problem
When invoking alarm control panel services: alarm_control_panel.alarm_arm_home, alarm_control_panel.alarm_disarm, etc.. T8400 device do not change their guard mode state. Other devices (T8441Z21) update as expected.

## Steps to Produce
1. Go to developer tools -> Services
2. Execute service with YAML as:
```
service: alarm_control_panel.alarm_arm_home
data: {}
target:
  device_id:<YOUR_DEVICE_ID>
```

## Expected Behavior
T8400 device guard mode should be updated to new state.

## Hardware Information about Home Assistant
Core version 2022.3.3
Operating System version 7.4

## Hardware Information about Cameras
- Camera Model T8400
- No base station

## Additional context
Looking at the logs I see that my T8441Z21 cameras log out additional response for `'event': 'property changed'` that the T8400 devices do not:

T8400:
```
2022-05-12 18:28:36 DEBUG (MainThread) [custom_components.eufy_security] eufy_security - WebSocket message sent. {"messageId": "set_guard_mode", "command": "station.set_guard_mode", "serialNumber": "<T8400_SERIAL>", "mode": 63}
2022-05-12 18:28:36 DEBUG (MainThread) [custom_components.eufy_security] eufy_security - on_message - {'type': 'result', 'success': True, 'messageId': 'set_guard_mode', 'result': {}}
```
T8441Z21:
```
2022-05-13 15:05:09 DEBUG (MainThread) [custom_components.eufy_security] eufy_security - WebSocket message sent. {"messageId": "set_guard_mode", "command": "station.set_guard_mode", "serialNumber": "<T8441Z21_SERIAL>", "mode": 1}
2022-05-13 15:05:09 DEBUG (MainThread) [custom_components.eufy_security] eufy_security - on_message - {'type': 'result', 'success': True, 'messageId': 'set_guard_mode', 'result': {}}
2022-05-13 15:05:10 DEBUG (MainThread) [custom_components.eufy_security] Manually updated eufy_security data
2022-05-13 15:05:11 DEBUG (MainThread) [custom_components.eufy_security] eufy_security - on_message - {'type': 'event', 'event': {'source': 'device', 'event': 'property changed', 'serialNumber': '<T8441Z21_SERIAL>', 'name': 'guardMode', 'value': 1, 'timestamp': 1652479511438}}
2022-05-13 15:05:11 DEBUG (MainThread) [custom_components.eufy_security] eufy_security - set_event_for_entity - device / <T8441Z21_SERIAL> / guardMode / 1
2022-05-13 15:05:11 DEBUG (MainThread) [custom_components.eufy_security] eufy_security - on_message - {'type': 'event', 'event': {'source': 'station', 'event': 'command result', 'serialNumber': '<T8441Z21_SERIAL>', 'command': 'set_guard_mode', 'returnCode': 0, 'returnCodeName': 'ERROR_PPCS_SUCCESSFUL'}}
2022-05-13 15:05:11 DEBUG (MainThread) [custom_components.eufy_security] Manually updated eufy_security data
2022-05-13 15:05:12 DEBUG (MainThread) [custom_components.eufy_security] eufy_security - on_message - {'type': 'event', 'event': {'source': 'station', 'event': 'guard mode changed', 'serialNumber': '<T8441Z21_SERIAL>', 'guardMode': 1}}
2022-05-13 15:05:12 DEBUG (MainThread) [custom_components.eufy_security] eufy_security - on_message - {'type': 'event', 'event': {'source': 'station', 'event': 'property changed', 'serialNumber': '<T8441Z21_SERIAL>', 'name': 'guardMode', 'value': 1, 'timestamp': 1652479511000}}
2022-05-13 15:05:12 DEBUG (MainThread) [custom_components.eufy_security] eufy_security - set_event_for_entity - station / <T8441Z21_SERIAL> / guardMode / 1
```

**Comment by tsvmitch** [comment]
I'm having an issue where the guard mode status will not update unless I reload the integration.

**Comment by shaunkutch** [comment]
I have tried reloading the integration as well. I have also tried removing and added the devices. The strange thing is that even after removing the cameras from the Eufy security app the devices still show up in the integration even after reloading or reinstalling the integration.

**Comment by shaunkutch** [comment]
After some more digging I found that I'm getting errors on the add-on side:
```
2022-05-17 07:01:44.546  INFO  Timeout connecting to station T8400xxxxxxxxxxx
2022-05-17 07:03:55.392  ERROR P2PClientProtocol.lookup(): station: T8400xxxxxxxxxxx - All address lookup tentatives 
```
Could be related to: https://github.com/fuatakgun/eufy_security/issues/326

**Comment by tsvmitch** [comment]
I too am getting similar errors. I just assumed it was because I don't have all of the cameras/sensors that are added to the Eufy account plugged in.

**Comment by eagleguts** [comment]
Just a heads up, I completely reinstalled my HA from scratch and still getting the same issue.

**Comment by tsvmitch** [comment]
I actually did that as well. I am running Hassio as a VM on my Unraid server.

**Comment by shaunkutch** [comment]
> I too am getting similar errors. I just assumed it was because I don't have all of the cameras/sensors that are added to the Eufy account plugged in. My home base will show the guard mode as armed. When I disarm it using the keypad, HA does not update. If I go to the "Eufy Integration" and click the three dots, then "reload", then go back to the home base device, it will show the current status. So I have to reload the integration ever time I change the guard mode.

This is what I'm experiencing as well. However I only see the status updated after restarting the add-on.

**Comment by shaunkutch** [comment]
This issue was resolved after updating Eufy Security Add-on to 0.9.1
